### PR TITLE
Runtasklistbugfixes

### PR DESCRIPTION
**Pull request by ritikaguptams**

### CODE REVIEW

File: src/App/EditPage.xaml.cs [hunk 1]
```diff
@@ -155,15 +155,58 @@ private async Task<bool> On_BackRequested()
...
+            }
+            DeleteFlyout.Hide();
+        }
+
+        private void BgDeleteButton_Click(object sender, RoutedEventArgs e)
+        {
+            var button = sender as Button;
+            activeTask = GetTaskFromButton(button);
+            activeTaskIndex = TasksCollection.IndexOf(activeTask);
+            activeTaskIsNowBg = true;
```
Comment (spaceisfun) [inline src/App/EditPage.xaml.cs:208]: nit these names should be activeTaskList* or activeList

**Review by spaceisfun — APPROVED** [review]

File: src/App/EditPage.xaml.cs (hunk 1 above)
Comment (ritikaguptams) [reply src/App/EditPage.xaml.cs:208]: Could you elaborate? Wouldn't it be the tasks within the tasklist that would be edited/deleted here.

Comment (spaceisfun) [reply src/App/EditPage.xaml.cs:208]: doh, you are right! :)

Comment (ritikaguptams) [reply src/App/EditPage.xaml.cs:208]: Aha! Thank you for the clarification.

**Pull request merged**